### DISCUSSION TITLE
Notify Status When Background Processing is Not Completed

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#13031] [Ideas]
Import and export processes can sometimes take a significant amount of time to complete. For instance, if three processes are registered and each takes 20 minutes, the current system does not send a completion notification until all three processes are finished. This is because a database notification job is issued only after each process is completed.
in this case, you would receive notifications for three items after one hour.

If a process takes longer than expected, users have no way to check the current status.

Therefore, it would be beneficial to notify users about the process status as soon as the background processing for import or export is created. This way, users can track the status of their processes.

Status: Before Start, In Progress, Completed (already implemented)

The images are samples of each notification: 
![スクリーンショット 2024-05-31 20 51 21](https://github.com/filamentphp/filament/assets/28666304/5cbffde7-d3da-4d97-871c-e40fedb6407f)
![スクリーンショット 2024-05-31 20 51 32](https://github.com/filamentphp/filament/assets/28666304/252e292b-468e-4874-b9a0-9499a448774a)
![スクリーンショット 2024-05-31 20 51 59](https://github.com/filamentphp/filament/assets/28666304/4a7a6af8-b5e7-4ef0-a383-8953f4123d11)

What are your thoughts on this idea? @danharrin  I would appreciate your feedback.

Thank you.

**Comment by danharrin** [comment]
Personally, I don't think this should be a feature in Filament directly. You could PR in some new events that you could then register listeners for if you want in your own app to achieve this?

Also, consider creating an ImportResource where users can track their import progress whenever they want

**Reply by bzy107** [reply]
Thank you for your response.

Currently, addressing this by creating an ImportResource.
If necessary, I will create a pull request for the listener.
Thank you very much.
